models/sap__0customer_attr.sql

```diff
@@ -0,0 +1,47 @@
+with kna1 as ( 
+
+	select * 
+	from {{ var('kna1') }}
+),
+
+final as (
+
+    select
+        kunnr,
+        brsch,
+        ktokd,
+        kukla,
+        land1,
+        lifnr,
+        loevm,
+        name1,
+        niels,
+        ort01,
+        pstlz,
+        regio,
+        counc,
+        stcd1,
+        stras,
+        telf1,
+        vbund,
+        bran1,
+        bran2,
+        bran3,
+        bran4,
+        bran5,
+        periv,
+        abrvw,
+        werks,
+        name2,
+        name3,
+        dear6,
+        pstl2
+    from kna1
+    {% if var('kna1_mandt_var',[]) %}
+	where mandt = '{{ var('kna1_mandt_var') }}'
+	{% endif %}
+    {{ dbt_utils.group_by(29) }}
```

Comment:
I see we are grouping by all the fields, but not aggregating any. Is this intentional?

Reply:
I think I was trying a unique test and it was failing without the group by. But I removed it because of the mandt question (see the conversation with Edwin).

Reply:
Added mandt back in and this test is now passing so removing the group by